src/projects.html

```diff
@@ -1,87 +1,143 @@
-            <a href="https://github.com/womenhackfornonprofits/pimp-my-cause-django" class="larger-button button">View Pimp My Cause Project</a>
-          </div>
+            <div class="project-img whfnp"></div>
+            <div class="project-info">
+                <h2 class="section-heading">Women Hack For Non-Profits</h2>
+                <h3>HTML5, CSS3, SASS, Grunt, AWS</h3>
+                <p>We recently completed an overhaul of our original
+                website in line with our rebrand, using modern tools and
+                standards to create an engaging and responsive user
+                experience.</p>
+                <p>We are currently working on powering it by a custom CMS
+                and as a future phase, integrate it with an API that
+                provides data surrounding our volunteers and
+                non-profits.</p><a class="larger-button button" href=
+                "https://github.com/womenhackfornonprofits/whfnp">View
+                WHFNP Project</a>
+            </div>
         </div>
-        <div class="project">
-          <div class="project-img ippr"></div>
-          <div class="project-info">
-            <h2 class="section-heading">IPPR</h2>
-            <h3> Express, CartoDb, Leaflet, ejs, jQuery</h3>
-            <p>Institute for Public Policy Research (IPPR) is an organisation with a mission to deliver independent, analytical, and critical research into social, political and economic issues.</p>
-            <p>The objective of our project with IPPR is to build a website to improve the transparency using open data. Our technology stack includes Node.js and Express on the backend, EmbeddedJs templates and jQuery on the front end, CartoDb's PostGIS database for data storage, data manipulation and analysis tools, and Leaflet for map visualisations.</p>
-            <a href="https://www.thematchbox.io" class="larger-button button">View IPPR Website</a>
-          </div>
-        </div>
-        <div class="project">
-          <div class="project-img whfnp"></div>
-          <div class="project-info">
-            <h2 class="section-heading">Women Hack For Non-Profits</h2>
-            <h3>HTML5, CSS3, SASS, Grunt, AWS</h3>
-            <p>We recently completed an overhaul of our original website in line with our rebrand, using modern tools and standards to create an engaging and responsive user experience.</p>
-            <p>We are currently working on powering it by a custom CMS and as a future phase, integrate it with an API that provides data surrounding our volunteers and non-profits.</p>
-            <a href="https://github.com/womenhackfornonprofits/whfnp" class="larger-button button">View WHFNP Project</a>
-          </div>
-        </div>
-      </div>
-    </section>
-    <section>
-      <div class="content text-section">
+    </div>
+</section>
+<section>
+    <div class="content text-section">
```

Comment:
should maybe `text-section` class goes on `<section>` as you did on rest of <section>'s